tests/unit/amo/pages/TestUserProfileEdit.js

```diff
@@ -197,36 +203,47 @@ describe(__filename, () => {
     // logged-in user (e.g., page refresh).
     renderUserProfileEdit({ errorHandler, params: {}, store });
 
-    sinon.assert.calledOnce(dispatchSpy);
+    // This should have been called twice. Once for fetchUserNotifications, and once for setViewContext.
+    sinon.assert.calledTwice(dispatchSpy);
     sinon.assert.calledWith(
       dispatchSpy,
       fetchUserNotifications({
         errorHandlerId: errorHandler.id,
         userId,
       }),
     );
+    sinon.assert.neverCalledWith(
+      dispatchSpy,
+      fetchUserAccount({
+        errorHandlerId: errorHandler.id,
+        userId,
+      }),
+    );
   });
 
-  it('does not dispatch any actions if the current logged-in user is being edited and the notifications are loaded', () => {
+  it('does not dispatch user actions if the current logged-in user is being edited and the notifications are loaded', () => {
     const userId = 2;
 
     const { store } = signInUserWithUserId(userId);
     const dispatchSpy = sinon.spy(store, 'dispatch');
 
-    store.dispatch(
-      loadUserNotifications({
-        userId,
-        notifications: createUserNotificationsResponse(),
-      }),
-    );
+    const loadUserNotificationsAction = loadUserNotifications({
+      userId,
+      notifications: createUserNotificationsResponse(),
+    });
+
+    store.dispatch(loadUserNotificationsAction);
 
     dispatchSpy.resetHistory();
 
     // This happens when loading the user edit profile page of the current
     // logged-in user (e.g., page refresh).
     renderUserProfileEdit({ params: {}, store });
 
-    sinon.assert.notCalled(dispatchSpy);
+    sinon.assert.neverCalledWithMatch(dispatchSpy, {
+      type: FETCH_USER_ACCOUNT,
+    });
+    sinon.assert.neverCalledWith(dispatchSpy, loadUserNotificationsAction);
```

Comment:
I don't think we want to assert that user notifications were not loaded, I think we want to assert they were not fetched.